src/app/_components/Selector/index.tsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+
+import { RefObject } from 'react';
+
+import Icon from '@/src/components/Icon';
+
+import useDropdown from './useDropdown';
+
+interface Item {
+  text: string;
+}
+
+interface SelectorProps {
+  items: Item[];
```

Comment:
```suggestion
interface SelectorProps {
  items: string[]
```
으로 될 수 있게 할 순 없을까요 ?? 
애초에 사용할 때 ['test1' , 'test2']와 같이 options를 넘겨줘도 좋을 것 같아요

Reply:
그냥 수정할 때는 에러가 발생했는데 사용법을 수정하면 가능할 것 같습니다.